test/executor.test.js

```diff
@@ -450,6 +450,38 @@ describe('executor', function() {
       });
     });
 
+    it('should parse simple config variable from env var', function(done) {
+      process.env.restApiRoot = '/url-from-env-var';
+      boot.execute(app, simpleMiddlewareConfig('routes',
+        { path: '${restApiRoot}' }
+      ));
+
+      supertest(app).get('/url-from-env-var').end(function(err, res) {
+        if (err) return done(err);
+        expect(res.body.path).to.equal('/url-from-env-var');
+        delete process.env.restApiRoot;
+        done();
+      });
+    });
+
+    it('dynamic variable from `env var` should have' +
+    ' precedence over app.get()', function(done) {
+      process.env.restApiRoot = '/url-from-env-var';
```

Comment:
That's a tough question that I don't have a good answer for. Let's use `afterEach` for these env-var cleanups then and ignore the other comment https://github.com/strongloop/loopback-boot/pull/177#discussion_r58389244